src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    humidity: 80,
+    wind: {
+      speed: 30,
+      direction: "s",
+    },
+    temperature: {
+      max: 12,
+      min: 12,
+    },
+  };
+
+  it("renders forecast details", () => {
+    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders correct values for props", () => {
+    const { getByTestId, getByText } = render(
+      <ForecastDetails forecast={validProps} />
+    );
+
+    expect(getByText("Monday, 30 April")).toHaveAttribute(
```

Comment:
Remember the idea behind React Testing Library. The idea is to test in a way that's relevant to the user. Here for example, I would query one of the temperature divs:
`const high = getByText("High")`
and then test based on the text content. Since your valid props has a max of 12, that's what I would check against in my assertion:
`expect(high).toHaveTextContent("High)`

So you would have a test like this:
`  it("max temperature shows correct value", () => {
    render(<ForecastDetails forecast={validProps} />);
    const high = screen.getByText("High", { exact: false });
    expect(high).toHaveTextContent("High: 12");
  });`